_posts/2022-10-06-audio-classification-torch/audio-classification.Rmd

```diff
@@ -0,0 +1,472 @@
+---
+title: "Audio classification with torch"
+description: >
+  Learn how to classify speech utterances with torch, making use of domain knowledge and deep learning. This post is a condensed version of the same-topic chapter in the book to be published with CRC Press, Deep Learning and Scientific Computing with R torch. 
+author:
+  - name: Sigrid Keydana
+    affiliation: RStudio
+    affiliation_url: https://www.rstudio.com/
+slug: keydanatorchaudio
+date: 2022-10-06
+categories:
+  - Torch
```

Comment:
I haven't checked, but is Torch always capitalized in the tags? I'm inferring that it's capitalized when referring to Torch (the library) versus small-t torch the R package (bonus confusion for the fact that the native package is also small-t it looks like from [the Wikipedia page](https://en.wikipedia.org/wiki/Torch_(machine_learning))).

I think it's fine either way—it's obvious from the fact that both Torch and R are in the tags that you're going to discuss the R package for it—just checking for consistency.

Reply:
it's just the category really that is capitalized (for aesthetical reasons, because it would look weird otherwise)